### PR TITLE
updated second link, first link still has paywall

### LINKED ISSUE #2767: Improve `skimage.feature.CENSURE` references
## Description
See https://github.com/scikit-image/scikit-image/blob/master/skimage/feature/censure.py#L148-L156 . The first reference is behind a pay-wall, the seconds one is broken.

### PR DESCRIPTION
**Pull request by SeraYang1**
## Description
Addressed second part of issue 2767 (feature/censure.py references), found link for second reference. Could not find a link for the first reference without a paywall.  

## References
Closes #2767.

## For reviewers
(Don't remove the checklist below.)

- [ ] Check that the PR title is short, concise, and will make sense 1 year
  later.
- [ ] Check that new functions are imported in corresponding `__init__.py`.
- [ ] Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.

### CODE REVIEW

File: skimage/feature/censure.py [hunk 1]
```diff
@@ -153,7 +153,7 @@ class CENSURE(FeatureDetector):
     .. [2] Adam Schmidt, Marek Kraft, Michal Fularz and Zuzanna Domagala
            "Comparative Assessment of Point Feature Detectors and
            Descriptors in the Context of Robot Navigation"
-           http://www.jamris.org/01_2013/saveas.php?QUEST=JAMRIS_No01_2013_P_11-20.pdf
+           http://yadda.icm.edu.pl/yadda/element/bwmeta1.element.baztech-268aaf28-0faf-4872-a4df-7e2e61cb364c/c/Schmidt_comparative.pdf
```
Comment (stefanv) [inline skimage/feature/censure.py]: Can you read this link?  It looks a bit more legit:

http://www.academia.edu/23315182/Comparative_assessment_of_point_feature_detectors_in_the_context_of_robot_navigation

Comment (stefanv) [inline skimage/feature/censure.py]: For #1, how about: http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.465.1117

Comment (SeraYang1) [inline skimage/feature/censure.py]: The academia.edu website seems to work, I can replace that. The first link shows the summary, but the download link seems to be broken. Should we use that anyways?

Comment (stefanv) [inline skimage/feature/censure.py]: When you say the download link seems to be broken, are you talking about the Citeseer link?  It works for me, right now.

Comment (soupault) [inline skimage/feature/censure.py]: For №1: I'd change the url to https://link.springer.com/chapter/10.1007/978-3-540-88693-8_8 and add `DOI:10.1007/978-3-540-88693-8_8` afterwards.

For №2: I'd keep the current link and add `DOI:10.1.1.465.1117`.

@stefanv I assume @SeraYang1 is speaking about link to .pdf, not to the page.

Comment (stefanv) [inline skimage/feature/censure.py]: Right, but I could also access the PDF.

Comment (SeraYang1) [inline skimage/feature/censure.py]: For #2 the current link before the PR?

Comment (soupault) [inline skimage/feature/censure.py]: @SeraYang1 no-no, sorry, the one you introduced.

Comment (SeraYang1) [inline skimage/feature/censure.py]: That link leads directly to a download, do I still need the DOI

Comment (soupault) [inline skimage/feature/censure.py]: Yes, .pdf is for convenience, DOI is for reference (FYI, we agreed to supplement with DOI as many links as possible).

Comment (SeraYang1) [inline skimage/feature/censure.py]: Alright! Thanks for the clarification!

**Pull request merged**